src/garage/tf/algos/_utils.py

```diff
@@ -0,0 +1,31 @@
+"""Utility functions used by tensorflow algorithms."""
+
+
+class _Default:  # pylint: disable=too-few-public-methods
+    """A wrapper class to represent default arguments.
+
+    Args:
+        val (object): Argument value.
+
+    """
+
+    def __init__(self, val):
+        self.val = val
+
+
+def make_optimizer(optimizer_type, **kwargs):
+    """Create an optimizer for tensorflow algos.
+
+    Args:
+        optimizer_type (type): Type of optimizer.
+        kwargs (dict): Other keyword arguments to initialize optimizer. This
+            is not used when `optimizer_type` is tuple.
```

Comment:
You didn't implement the case when `optimizer_type` is a tuple.

Here is the use case - `optimizer_type` will be specified by the user, and `kwargs` is provided by the algorithm as a set of sane defaults. User can override default by passing `optimizer_type` as an optimizer type and a parameter dict. If two parameters of the same name appear in both `optimizer_type` and `kwargs`, take the one in `optimizer_type`.